app/views/shared/_back_to_login.html.erb

```diff
@@ -0,0 +1,3 @@
+<div data-go-back-link='true' class='go-back-link' style='display:none;'>
+  <%= link_to "‹ Go back (show the login option)", 'javascript:;', class: "#{content_column_class} #{label_column_offset_class}", data: { no_sunet: true, target: '[data-no-sunet-content]', content_target: '[data-no-sunet-target]', link: '[data-go-back-link]' } %>
```

Comment:
This is done. 

> On May 4, 2015, at 9:43 AM, Jessie Keck <notifications@github.com> wrote:
> 
> In app/views/shared/_back_to_login.html.erb <https://github.com/sul-dlss/sul-requests/pull/107#discussion_r29600090>:
> 
> > @@ -0,0 +1,3 @@
> > +<div data-go-back-link='true' class='go-back-link' style='display:none;'>
> > +  <%= link_to "‹ Go back (show the login option)", 'javascript:;', class: "#{content_column_class} #{label_column_offset_class}", data: { no_sunet: true, target: '[data-no-sunet-content]', content_target: '[data-no-sunet-target]', link: '[data-go-back-link]' } %>
> @jgreben <https://github.com/jgreben> just FYI, there was another comment on this file that was removed because the line was updated, but there is still one change to make.
> 
> The < in the link text is not correct. See @jvine <https://github.com/jvine>'s comment at #107 (comment) <https://github.com/sul-dlss/sul-requests/pull/107#discussion_r29518915>
> —
> Reply to this email directly or view it on GitHub <https://github.com/sul-dlss/sul-requests/pull/107/files#r29600090>.
>